styles.css

```diff
@@ -382,3 +383,401 @@ body {
   justify-content: space-between;
   width: 90%;
 }
+
+.your-email,
+.your-name,
+.your-message {
+  color: transparent;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-bar {
+    width: 100%;
+    padding-top: 50px;
+    padding-bottom: 20px;
+    background-color: #fff;
+    box-shadow: 0 0.5px 0 #dddfe6;
+    padding-left: 35%;
+  }
+
+  .homepage-heading {
+    font-size: 110px;
+    margin-top: 132px;
+    margin-left: 160px;
+  }
+
+  .homepage-description1,
+  .homepage-description2 {
+    margin-left: 160px;
+    width: 484px;
+  }
+
+  .homepage-start-collab {
+    margin-left: 140px;
+    width: 254px;
+    border: none;
+  }
+
+  .homepage-container-img {
+    transform: rotate(360deg);
+    margin-left: 815px;
+    margin-top: -668px;
+    height: 877px;
+    width: 702px;
+  }
+
+  .dotted-square {
+    background-image: url(images/dotted-square.png);
+    background-repeat: no-repeat;
+    height: 200px;
+    width: 300px;
+    position: absolute;
+    margin-left: 540px;
+    margin-top: -205px;
+  }
+
+  .projects {
+    margin-top: 400px;
+  }
+
+  .projects-heading {
+    position: absolute;
+    font-size: 110px;
+    transform: rotate(-90deg);
+    margin-left: 150px;
+  }
+
+  .hamburger .line,
+  .image,
+  .hamburger,
+  .nav-img,
+  #project2,
+  #project3,
+  #project4,
+  #project5 {
+    display: none;
+  }
+
+  .hamburger-menu {
+    display: contents;
+  }
+
+  .hamburger-menu ul {
+    list-style: none;
+    display: flex;
+  }
+
+  .see a {
+    color: #1a2236;
+  }
+
+  .hamburger-menu li a {
+    text-decoration: none;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    color: #1a2236;
+    margin-left: 40px;
+  }
+
+  .projects-project1 {
+    width: 852px;
+    margin-top: -90px;
+    height: 490px;
+    margin-left: 334px;
+  }
+
+  .dotted1 {
+    position: absolute;
+    background: url(images/dotted-square.png);
+    background-repeat: no-repeat;
+    margin-left: 90px;
+    margin-top: 179px;
+    height: 140px;
+    width: 170px;
+  }
+
+  .projects-project2 {
+    position: absolute;
+    width: 465px;
+    height: 270px;
+    margin-top: 6px;
+  }
+
+  .projects-project3 {
+    position: absolute;
+    width: 830px;
+    height: 465px;
+    margin-top: 138px;
+  }
+
+  .projects-project4 {
+    position: absolute;
```

Comment:
- [Optional] I noticed you used position property a lot in your CSS. I will suggest you use other CSS3 properties like flexbox for laying out contents over positioning. The position property is not efficient for responsiveness. Kindly consider refactoring to use flexbox.